### PR TITLE
[BA-479]: including blockhound configuration to the starter (RC 0.1.5)

### PR DESCRIPTION
**Pull request by vlad24**
Now all dependents will automatically do without extra blockhound setup

### CODE REVIEW

File: starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    @Override
+    public void applyTo(BlockHound.Builder builder) {
+        builder
+                .allowBlockingCallsInside(
+                        org.springdoc.core.OpenAPIService.class.getName(),
+                        "build"
+                );
+    }
+
```
Comment (mgrabul) [inline starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java]: remove line

File: starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.vivy.springdoc;
+
+import com.google.auto.service.AutoService;
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+
+@AutoService(BlockHoundIntegration.class)
+public class SwaggerBlockHoundIntegration implements BlockHoundIntegration {
```
Comment (mgrabul) [inline starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java:9]: do you require a test?

Comment (vlad24) [reply starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java:9]: not really, this is instruction for other dependents

File: starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java (hunk 1 above)
Comment (vlad24) [reply starter/src/main/java/com/vivy/springdoc/SwaggerBlockHoundIntegration.java]: done

**Review by mgrabul — APPROVED** [review]

**Pull request merged**